controllers/suite_test.go

```diff
@@ -0,0 +1,83 @@
+/*
```

Comment:
this file was not here before I think we can remove this for now and work on unit test later